test/sim/moves/hyperbeam.js

```diff
@@ -42,13 +42,13 @@ describe(`Hyper Beam`, function () {
 		assert.false.cantMove(() => battle.choose('p1', 'move tackle'));
 	});
 
-	it.skip(`[Gen 1] should force a recharge turn after damaging, but not breaking a Substitute`, function () {
+	it(`[Gen 1] should force a recharge turn after damaging, but not breaking a Substitute`, function () {
 		battle = common.gen(1).createBattle([[
-			{species: 'chansey', moves: ['hyperbeam', 'tackle']},
+			{species: 'slowbro', moves: ['hyperbeam', 'tackle']},
```

Comment:
Why is this changed? Are we sure that Slowbro's Hyper Beam isn't just breaking the substitute?

Reply:
Chansey is faster than Rhydon so the previous test was inappropriate. We want to test hitting the Substitute, not the Rhydon itself. Slowbro is slower, and also never breaks Rhydon's sub even with a max roll crit.

Reply:
Gotcha.